src/snowflake/connector/aio/_network.py

```diff
@@ -827,6 +827,10 @@ async def _request_exec(
             RuntimeError,
             AttributeError,  # json decoding error
         ) as err:
+            if isinstance(err, RuntimeError) and "Event loop is closed" in str(err):
+                logger.info(
+                    "If you see the logging error message 'RuntimeError: Event loop is closed' during program exit, it probably indicates that the connection was not closed properly before the event loop was shut down. Please use SnowflakeConnection.close() to close connection."
+                )
```

Comment:
do we need to re-raise error here or it doesn't matter?
otherwise the error gets swallowed, and _request_exec returns None, which will be an unknown error handled by the retry logic?